### DISCUSSION TITLE
Suggestions to make crowds of mobs a bigger threat

### DISCUSSION THREAD

**Discussion by Geonaut3** [sf-inc/sihywtcamd#24] [Ideas]
Sometimes certain mobs could seek other mobs to form a group, excluding ranged mobs such as skeletons because they could end up killing each other in crowds. This could happen by having only specific mobs have the ability to form a crowd when they  see each other, while others don't have the ability and don't form crowds.
I-frames could also be modified to allow attacks from different mobs to hit at the same time, meaning that if you are trapped in a crowd you will die very quickly.
Paired with your combat update mod this would make the sweeping edge enchantment more important.

**Comment by charlyb01** [comment]
I like the idea of crowd a lot! However it seems quite tricky to implement... As the idea seems really nice, I will make some tries but it may take a while. Anyway, thanks for the idea! =)
PS: But I don't think I will change the I-frames because it could be too much

**Comment by Geonaut3** [comment]
That's great that you're considering the crowding idea! The i-frames part is probably outside of the scope of the mod anyways.
